nb/projects/belt_sprockets/belt_sprockets.nb

```diff
@@ -0,0 +1,683 @@
+//Ported to JSxCAD from https://www.thingiverse.com/thing:16627/files
+
+// Parametric Pulley with multiple belt profiles
+// by droftarts January 2012
+
+// Based on pulleys by:
+// http://www.thingiverse.com/thing:11256 by me!
+// https://github.com/prusajr/PrusaMendel by Josef Prusa
+// http://www.thingiverse.com/thing:3104 by GilesBathgate
+// http://www.thingiverse.com/thing:2079 by nophead
+
+// dxf tooth data from http://oem.cadregister.com/asp/PPOW_Entry.asp?company=915217&elementID=07807803/METRIC/URETH/WV0025/F
+// pulley diameter checked and modeled from data at http://www.sdp-si.com/D265/HTML/D265T016.html
+
+// tune-able constants
+
+const makePulley = (profile, teeth) => {
+  // Number of teeth, standard Mendel T5 belt = 8, gives Outside Diameter of 11.88mm
+  // Profiles: 1=MXL 2=40DP 3=XL 4=H 5=T2.5 6=T5 7=T10 8=AT5 9=HTD_3mm 10=HTD_5mm 11=HTD_8mm 12=GT2_2mm 13=GT2_3mm 14=GT2_5mm
+
+  //	********************************
+  //	** Scaling tooth for good fit **
+  //	********************************
+  /*	To improve fit of belt to pulley, set the following constant. Decrease or increase by 0.1mm at a time. We are modelling the *BELT* tooth here, not the tooth on the pulley. Increasing the number will *decrease* the pulley tooth size. Increasing the tooth width will also scale proportionately the tooth depth, to maintain the shape of the tooth, and increase how far into the pulley the tooth is indented. Can be negative */
+
+  const additional_tooth_width = 0.2; //mm
+
+  //	If you need more tooth depth than this provides, adjust the following constant. However, this will cause the shape of the tooth to change.
+
+  const additional_tooth_depth = 0; //mm
+
+  // Functions
+
+  const tooth_spaceing_curvefit = (b, c, d) => {
```

Comment:
How about ToothSpacingCurvefit ?

Reply:
I've been doing toothSpacingCurvefit, when do you do a capital first letter and when do you do lowercase?

Reply:
Ah, as usual you already answered this :p "I've been using CamelCase for functions and camelCase for variables."